packages-common/pluggable-widgets-tools/tests/commands.spec.js

```diff
@@ -33,79 +37,105 @@ describe("pluggable-widgets-tools commands", () => {
         rm(toolsPackagePath);
     });
 
-    describe.each(CONFIGS)("for %s %s widget created with generator %s", (platform, lang, version) => {
-        const widgetName = `generated_${version.replace(".", "_")}_${lang}_${platform}`;
+    describe.each(CONFIGS)("for %s %s widget created with generator %s", (platform, boilerplate, lang, version) => {
```

Comment:
🎄  `boilerplate` is usable only for `native` => maybe we should express it somehow differently?

Reply:
What do you mean exactly? Boilerplate is also used for web.